R/run.R

```diff
@@ -200,20 +216,39 @@ run_bm <- function(bm, ..., n_iter = 1, profiling = FALSE, global_params = list(
 
   metadata <- assemble_metadata(
     name = bm$name,
-    params = params,
+    params = bm$tags_fun(params),
     cpu_count = global_params$cpu_count,
     n_iter = n_iter
   )
 
   out <- BenchmarkResult$new(
-    name = bm$name,
-    result = do.call(rbind, results),
-    params = all_params,
+    run_name = NULL,
+    run_id = NULL,
+    batch_id = batch_id,
+    run_reason = NULL,
+    # timestamp = utc_now_iso_format(),  # let default populate
```

Comment:
Do we need this comment?

Reply:
Not really, I just left it so you can see all the fields on the object here even if they're not getting filled in here. It's a little confusing to put it at the end of the line as it's really why the line is commented out; I could put it as a line before, or just drop it all. Or we could actually specify timestamp here. It's all minor.